api/base/views.py

```diff
@@ -464,11 +463,13 @@ def root(request, format=None):
         'links': {
             'nodes': absolute_reverse('nodes:node-list'),
             'users': absolute_reverse('users:user-list'),
+            'collections': absolute_reverse('collections:collection-list'),
+            'registrations': absolute_reverse('registrations:registration-list'),
+            'institutions': absolute_reverse('institutions:institution-list'),
+            'licenses': absolute_reverse('licenses:license-list'),
+            'metaschemas': absolute_reverse('metaschemas:metaschema-list'),
```

Comment:
It looks like the feature that uses this will remain in dev_mode for the moment, so could you move metaschemas down into the if block you deleted below? Everything else looks right. Pass done. 🐧